lib/pymedphys/docs/background/file-structure.rst

```diff
@@ -1,10 +1,6 @@
 Repository File Structure
 =========================
 
-.. contents::
-    :local:
-    :backlinks: entry
-
 The PyMedPhys repository has the following general structure:
 
 .. code-block:: bash
```

Comment:
the repository structure has changed.
pymedphys/lib/pymedphys/ is now the starting point, including docs.
This is a "must fix"

Reply:
Yup, good call. This is a very helpful file... I wish it could be autogenerated :/...